website/docs/r/notification_destination.html.markdown

```diff
@@ -223,16 +223,21 @@ resource "newrelic_notification_destination" "foo" {
 
 #### [Slack](https://docs.newrelic.com/docs/alerts-applied-intelligence/notifications/notification-integrations/#slack)
 
-Slack destinations are created by actively interacting with the UI. This is how OAuth authentication works and there is no way to authenticate otherwise. Because of this, there is no way to create a slack destination via terraform, and must be created in the UI and imported into terraform.
+Slack destinations are created by actively interacting with the UI. This is how OAuth authentication works and there is no way to authenticate otherwise. Because of this, there is no way to create a slack destination via terraform, and must be created in the UI and imported into terraform. More information in the [Import](#import) section below.
```

Comment:
I'm thinking of also stating that slack destinations are best used as data sources and not as resources, wdyt?

Reply:
I know that you did not change this part, but there's a typo in "and must be created" (should be "and it must be created")

Frankly, I would rephrase this whole part to something like. OAuth part does not really matter.

> In order to create a Slack destinations, you have to go grant our application access to your workspace. This process is [based on OAuth](https://api.slack.com/authentication/oauth-v2) can only be done through a browser. 
As a result, you cannot set up a Slack destination purely with Terraform code. 
You can either create a Slack destination in the interface and [import](#import) it to Terraform, or use destination data source.

TODO: add a link to the destination data source page

Reply:
👍